app.py

```diff
@@ -26,19 +26,49 @@ async def startup():
     loop.create_task(client.start(TOKEN))
 
 
-@server.get('/users/{user_id}/roles')
+@server.get("/users/{user_id}/roles")
 async def test_endpoint(user_id: int) -> dict:
     """Get a user's roles."""
     guild = client.get_guild(GUILD_ID)
     try:
         member = await guild.fetch_member(user_id)
     except discord.NotFound:
-        raise fastapi.HTTPException(status_code=404, detail='User not found.')
+        raise fastapi.HTTPException(status_code=404, detail="User not found.")
     return {
-        'roles': [
+        "roles": [
+            {"name": role.name, "id": role.id}
+            for role in member.roles
+            if not role.is_default()
+        ]
+    }
+
+
+@server.get("/bans")
+async def ban_endpoint():
+    """Get a guilds bans."""
+    guild = client.get_guild(GUILD_ID)
+    try:
+        ban_list = []
+        bans = await guild.bans()
+        for ban in bans:
+            ban_list.append(ban)
+
+    except discord.errors.Forbidden:
+        raise fastapi.HTTPException(status_code=403, detail="Missing Permissions")
+    
+    return {
+        "bans": [
             {
-                'name': role.name,
-                'id': role.id
-            } for role in member.roles if not role.is_default()
+                "user": [
+                    {
+                        "userId": banEntry.user.id,
+                        "userName": banEntry.user.name,
+                        "userDiscriminator": banEntry.user.discriminator,
+                        "userIsBot": banEntry.user.bot,
+                    }
+                ],
+                "reason": banEntry.reason,
+            }
+            for banEntry in ban_list
```

Comment:
```suggestion
            for ban_entry in ban_list
```
And *definitely* snake case for Python.